data_science/terraform/modules/dlami/variables.tf

```diff
@@ -1,62 +1,33 @@
-variable "name" {
-  description = "Name of the ASG to create"
-}
+variable "name" {}
+variable "instance_type" {}
 
-variable "subnet_list" {
-  type = "list"
+variable "hashed_password" {
+  default = "sha1:5310f21e370d:a4d66e725c179218638c21c03d83933aa066db2d"
 }
 
-variable "key_name" {
-  description = "SSH key pair name for instance sign-in"
+variable "bucket_name" {
+  description = "Bucket for storing Jupyter notebooks with s3contents plugin"
 }
 
-variable "vpc_id" {
-  description = "VPC for EC2 autoscaling group security group"
+variable "default_environment" {
+  description = "Python environment to install modules in"
 }
 
+variable "vpc_id" {}
+variable "custom_security_groups" {
+  type        = "list"
+  default     = []
+}
 variable "controlled_access_cidr_ingress" {
   type        = "list"
   default     = []
   description = "CIDR for SSH access to EC2 instances"
 }
-
-variable "custom_security_groups" {
-  type    = "list"
-  default = []
+variable "key_name" {}
+variable "ebs_volume_size" {
+  default = 250
 }
```

Comment:
Why have `vpc_id` and `key_name` variables?
Could make it frustrating to have to do each time.